app/javascript/src/components/Authentication/Signup.jsx

```diff
@@ -43,67 +35,63 @@ const Signup = ({ history }) => {
-          onSubmit={onSubmit}
-          validationSchema={formValidationSchemas.signupForm}
+          onSubmit={handleSubmit}
+          validationSchema={SIGNUP_FORM_VALIDATION_SCHEMA}
         >
-          {({ handleSubmit }) => (
-            <Form className="w-full p-8 space-y-6 bg-white border rounded-md shadow">
-              <Input
-                name="email"
-                type="email"
-                label="Email"
-                placeholder="oliver@example.com"
-                required
-              />
-              <Input
-                name="firstName"
-                type="text"
-                label="First name"
-                placeholder="Sam"
-                required
-              />
-              <Input
-                name="lastName"
-                type="text"
-                placeholder="Smith"
-                label="Last name"
-                required
-              />
-              <Input
-                name="password"
-                type="password"
-                label="Password"
-                placeholder="******"
-                required
-              />
-              <Input
-                name="passwordConfirmation"
-                type="password"
-                label="Confirm password"
-                placeholder="******"
-                required
-              />
-              <Button
-                type="submit"
-                onClick={e => {
-                  e.preventDefault();
-                  setSubmitted(true);
-                  handleSubmit();
-                }}
-                className="h-8"
-                loading={loading}
-                label="Signup"
-                fullWidth
-              />
-            </Form>
-          )}
+          <Form className="w-full p-8 space-y-6 bg-white border rounded-md shadow">
+            <Input
+              required
+              name="email"
+              type="email"
+              label="Email"
+              placeholder="oliver@example.com"
+            />
+            <Input
+              required
+              name="firstName"
+              type="text"
+              label="First name"
+              placeholder="Sam"
+            />
+            <Input
+              required
+              name="lastName"
+              type="text"
+              placeholder="Smith"
+              label="Last name"
+            />
+            <Input
+              required
+              name="password"
+              type="password"
+              label="Password"
+              placeholder="******"
+            />
+            <Input
+              required
+              name="passwordConfirmation"
+              type="password"
+              label="Confirm password"
+              placeholder="******"
+            />
+            <Button
+              fullWidth
+              type="submit"
+              onClick={() => {
+                setSubmitted(true);
+              }}
```

Comment:
Not sure why you haven't made this change although pointed out in review comments.
```suggestion
              onClick={() => setSubmitted(true)}
```